include/erlmld.hrl

```diff
@@ -8,9 +8,13 @@
           %% record sub-sequence number for records using KPL aggregation:
           sub :: undefined | non_neg_integer(),
 
+          %% record sub-sequence number for records NOT using KPL aggregation.
```

Comment:
May want to elaborate that this is also for the KPL-like aggregation used/supported by erlmld.